director/lib/template_context_processors.py

```diff
@@ -1,26 +1,34 @@
+"""
+Functions for defining template rendering contexts.
+
+Each function defines variables that will be available to be
+used in templates. Grouped into functions by type of variable.
+Prefer adding a variable to an existing function, rather than
+creating a new function. If you do create a function make sure
+to add it to `TEMPLATES.OPTIONS.context_processors` in `settings.py`.
+"""
 import typing
 
-from django.conf import settings
+from django import conf
```

Comment:
This is fine and it works, but it's just different then everywhere else in the project where we do `from django.conf import settings` as it was before.

Reply:
Yep, I know but `settings` clashed with `def settings`. Should I change it to `from django.conf import settings as django_settings` to be a little more consistent while avoiding the clash.

Reply:
Oh true. Nah don't worry about it.